.github/ISSUE_TEMPLATE/pull_request.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Pull request
+about: Use this template to open pull requests
+title: '<title>'
+labels:
+assignees: 'KateFriedman-NOAA', 'WalterKolczynski-NOAA'
+
+---
+
```

Comment:
Before the description, add directions along these lines:

Before opening a PR, please note these guidelines:
- Each PR should only address ONE topic and have an associated issue
- No hardcoded or paths to personal directories should be present
- No temporary or backup files should be committed
- Any code that was disabled by being commented out should be removed

Reply:
Will do.